core/driver/sources/src/main/scala/org/virtuslab/ideprobe/reporting/ErrorValidator.scala

```diff
@@ -18,8 +20,13 @@ object ErrorValidator {
     }
   }
 
+  private def ideMessageMatchesMessagesFromConfig(ideMessage: IdeMessage, messagesFromConfig: Seq[String]): Boolean =
+    messagesFromConfig.exists(configMessage => trimMultiLine(ideMessage.content).contains(trimMultiLine(configMessage)))
+
+  private def trimMultiLine(s: String): String = s.split('\n').map(_.trim).mkString("\n")
```

Comment:
I'd call it
```suggestion
  private def trimEachLine(s: String): String = s.split('\n').map(_.trim).mkString("\n")
```
As `trimMultiLine` might suggest the the **entire multiline string** is trimmed, rather than each individual line

Reply:
Wow btw apostrophe works in Scala? 😯 I thought it was only confined to [Symbols](https://www.scala-lang.org/api/2.13.6/scala/Symbol.html) and generally deprecated

Reply:
Hm, here we have just a plain `Char` usage - `s.split('\n')` ... I don't get it?

Reply:
Aww crap, sorry, I get JVM languages confused ;p sure, all good